packages/dashboard/src/customization/widgets/status/component.css

```diff
@@ -0,0 +1,16 @@
+.status-widget-empty-state {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 0 2em;
```

Comment:
Can we wrap the widget in a box to handle the padding? I want to defer all of the spacing styles to cloudscape so it's consistent.